installer/install.iss

```diff
@@ -2692,6 +2699,32 @@ begin
         LogError(ExpandConstant('Line {#__LINE__}: Unable to remove the Git for Windows updater (output: '+ReadFileAsString(LogPath)+', errors: '+ReadFileAsString(ErrPath)+').'));
 end;
 
+procedure InstallWindowsTerminalFragment;
+var
+    Res:Longint;
+    AppPath,JSONPath:String;
+begin
+    if IsAdminInstallMode() then
+        JSONPath:=ExpandConstant('{commonappdata}\Microsoft\Windows Terminal\Fragments\Git\git-bash.json')
+    else
+        JSONPath:=ExpandConstant('{localappdata}\Microsoft\Windows Terminal\Fragments\Git\git-bash.json');
+    AppPath:=ExpandConstant('{app}');
+    StringChangeEx(AppPath, '\', '/', True)
+    if not SaveStringToFile(JSONPath,
```

Comment:
If the `{AppData}\Microsoft\Windows Terminal\Fragments\Git` path doesn't exist, then the json file won't be written. Better to make sure that the path exists and make one if not.

Testing the latest released version as of now, the installer shows the following error…

```log
Unable to install Windows Terminal Fragment to C:\ProgramData\Microsoft\Windows Terminal\Fragments\Git\git-bash.json
```

Reply:
@dscho

Reply:
We do create the directory before writing a file to it, but there was an oversight with upgrade installations. See git-for-windows/git#3312 for details and #363 for the proposed fix.